types/plotly.js/index.d.ts

```diff
@@ -14,6 +14,7 @@
 //                 Pramod Mathai  <https://github.com/skippercool>
 //                 Takafumi Yamaguchi <https://github.com/zeroyoichihachi>
 //                 Michael Adams <https://github.com/mtadams007>
+// 								 Archit Bhatia <https://github.com/abhatia-lix>
```

Comment:
Spacing is inconsistent here (use space instead of tabs to match with the rest of the file).